osm2gtfs/creators/routes_creator.py

```diff
@@ -13,4 +13,16 @@ def __repr__(self):
         return rep
 
     def add_routes_to_schedule(self, schedule, data):
+        """
+        route_id  # Required: From Line
+        route_type  # Required: From Line
+
+        route_short_name  # Required: To be generated from Line or Itinerary
+        route_long_name  # Required: To be generated from Line or Itinerary
+
+        route_desc # From Line
+        route_url  # From Line
+        route_color # From Line
+        route_text_color  # From Line
+        """
```

Comment:
What do these comments mean? What do they relate to?

Reply:
They were just to not forget important values :smile: Can be taken out.